docs/tutorial.md

```diff
@@ -19,7 +19,13 @@ To successfully complete this tutorial you must:
 * Install the [New Relic Infrastructure Agent](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/installation/install-infrastructure-linux)
 
 ## Building the structure of the integration
-**Step1: Install nr-integrations-builder**
+**Step1:** Create the directory where you want to place the integrations (it needs to be under `$GOPATH/src`)
```

Comment:
This Step1 is redundant with the Step 1 from the next section. So we think that it's better to not include it here. 
We've realised that this section has some more redundant elements. We will refactor it to not confuse our users.
Would you agree to remove this part?